lighthouse-core/gather/driver.js

```diff
@@ -286,18 +287,23 @@ class Driver {
       }
     }
     return new Promise(async (resolve, reject) => {
-      const asyncTimeout = setTimeout((_ => {
-        const err = new LHError(LHError.errors.PROTOCOL_TIMEOUT);
-        err.message += ` Method: ${method}`;
-        reject(err);
-      }), timeout);
+      let asyncTimeout;
+      if (timeout > 0) {
```

Comment:
Taking another look at this.
I'm a bit hesitant to absorb this complexity.

What about adding a driver.innerSendCommand() which does the essence of sendCommand
And for the Page.Navigate command we use that.

Reply:
Done